### ISSUE TITLE
Aliens vs. Predator (10680)

### ISSUE THREAD

**Issue by BLaDZer** [ValveSoftware/Proton#530]
```
System:    Host: blad-pc Kernel: 4.18.3-1-MANJARO x86_64 bits: 64 Desktop: Xfce 4.12.4 
           Distro: Manjaro Linux 
CPU:       Topology: 6-Core model: AMD FX-6100 bits: 64 type: MCP L2 cache: 2048 KiB 
           Speed: 1541 MHz min/max: 1400/3300 MHz Core speeds (MHz): 1: 1554 2: 1806 3: 1527 4: 1529 
           5: 1535 6: 1529 
Graphics:  Card-1: AMD Pitcairn PRO [Radeon HD 7850 / R7 265 / R9 270 1024SP] driver: amdgpu v: kernel 
           Display: x11 server: X.Org 1.20.1 driver: amdgpu unloaded: modesetting 
           resolution: 1360x768~60Hz 
           OpenGL: 
           renderer: AMD Radeon HD 7800 Series (PITCAIRN DRM 3.26.0 4.18.3-1-MANJARO LLVM 6.0.1) 
           v: 4.5 Mesa 18.1.6
```
game won't start (without any windows)
[steam-10680.log](https://github.com/ValveSoftware/Proton/files/2320627/steam-10680.log)

**Comment by Corben78** [comment]
This is strange, I got it running once. Can't reproduce it now.

On first start it wants to open a ' readme.txt' (with a leading space) file which doesn't exist and would normally be the 'Readme.txt' file in the games folder. After letting proton's notepad create the file, I was able to start and play the game, DX9 as well as DX11. It had crackling sound in the menue as well as in game, but not in the cutscenes. While playtesting it crashed in Singleplayer when activating the first powerswitch. Since then I was not able to start the game again. I verified the game cache and cleaned the prefix, but no luck so far.

I'm wondering why it's so stubborn with proton as it's working flawless (except DX11) in vanilla wine (platinum rating).

**Comment by Corben78** [comment]
I tinkered around a bit further. The audio issue seems related to the wine version.
With wine stable (3.0.2 atm) it works as platinum rating. With wine 3.7, 3.10 or 3.14 I can start it always, but have audio issues. Cutscenes are fine, but in the menu and in game have crackling or no audio.

Proton still refuses to start the game, though it's wine 3.7

**Comment by jtlayton** [comment]
Fails to start for me too under proton.

```
Computer Information:
    Manufacturer:  Unknown
    Model:  Unknown
    Form Factor: Desktop
    No Touch Input Detected

Processor Information:
    CPU Vendor:  AuthenticAMD
    CPU Brand:  AMD Ryzen 7 1700 Eight-Core Processor          
    CPU Family:  0x17
    CPU Model:  0x1
    CPU Stepping:  0x1
    CPU Type:  0x0
    Speed:  3000 Mhz
    16 logical processors
    8 physical processors
    HyperThreading:  Supported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Supported
    SSE41:  Supported
    SSE42:  Supported
    AES:  Supported
    AVX:  Supported
    CMPXCHG16B:  Supported
    LAHF/SAHF:  Supported
    PrefetchW:  Unsupported

Operating System Version:
    "Fedora release 28 (Twenty Eight)" (64 bit)
    Kernel Name:  Linux
    Kernel Version:  4.17.18-200.fc28.x86_64
    X Server Vendor:  Fedora Project
    X Server Release:  11906000
    X Window Manager:  GNOME Shell
    Steam Runtime Version:  steam-runtime-beta-release_2018-06-14

Video Card:
    Driver:  NVIDIA Corporation GeForce GTX 1060 6GB/PCIe/SSE2
    Driver Version:  4.6.0 NVIDIA 396.45
    OpenGL Version: 4.6
    Desktop Color Depth: 24 bits per pixel
    Monitor Refresh Rate: 59 Hz
    VendorID:  0x10de
    DeviceID:  0x1c03
    Revision Not Detected
    Number of Monitors:  2
    Number of Logical Video Cards:  1
    Primary Display Resolution:  3440 x 1440
    Desktop Resolution: 5360 x 1440
    Primary Display Size: 31.42" x 13.19" (34.06" diag)
                                            79.8cm x 33.5cm (86.5cm diag)
    Primary Bus: PCI Express 16x
    Primary VRAM: 6144 MB
    Supported MSAA Modes:  2x 4x 8x 16x 

Sound card:
    Audio device: USB Mixer

Memory:
    RAM:  64301 Mb

Miscellaneous:
    UI Language:  English
    LANG:  en_US.UTF-8
    Total Hard Disk Space Available:  856614 Mb
    Largest Free Hard Disk Block:  115243 Mb
    VR Headset: None detected

Recent Failure Reports:
```


[steam-10680.log](https://github.com/ValveSoftware/Proton/files/2340134/steam-10680.log)

**Comment by Corben78** [comment]
This is due to CEG DRM, see https://github.com/ValveSoftware/Proton/issues/753

I discovered this, when tinkering further around. As I didn't want to re-download the game again and again, I copied the files from my working wine 3.0.3 prefix. Here the binaries are working, and suddenly I was able to start the game with proton. As I already reported in my previous post above, when I didn't understand why. When verifying the integrity of the files, it'll overwrite the executables and it won't start anymore.

I also tried to download the windows version via steamcmd and let it install into my steamapp directory, but here the binaries also don't work.
```bash
steamcmd +@sSteamCmdForcePlatformType windows +login <mysteamlogin> +force_install_dir "~/.local/share/Steam/steamapps/common/Aliens\ vs\ Predator/" +app_update 10680 install +quit
```

With the working binary the game crashes after a while with proton. In marine single player as soon as you pull the switch after you found the dead worker, but also in multiplayer. I guess it's related to an audio issue.
In the DX11 version I have no audio at all, DX9 version does have audio. Both crash though at the same situation.
I tried to install xact:
```bash
WINEPREFIX=~/.local/share/Steam/steamapps/compatdata/10680/pfx WINE=~/.local/share/Steam/steamapps/common/Proton\ 3.7/dist/bin/wine64 winetricks
```
And set xaudio2_5 and xaudio2_7 to native (tried POL's wine 3.7 so it wouldn't reconfigure the prefix, but it still does, which also doesn't seem to matter):
```bash
WINEPREFIX=~/.local/share/Steam/steamapps/compatdata/10680/pfx ~/.PlayOnLinux/wine/linux-amd64/3.7/bin/winecfg
```
But it didn't change anything, DX9 has sound, DX11 doesn't, both versions still crash. Still the DX9 version in wine stable (3.0.3 atm) has platinum rating and works flawless.

FWIW I was able to get rid of the audio crackling with the help of [Optimize PulseAudio for Steam](https://ludwig.im/en/projects/steam-pulseaudio-sound-latency-lagging-problem-noise).
In short: edit `/etc/pulse/daemon.conf`:
```
# Optimized fragements for steam
default-fragments = 5
default-fragment-size-msec = 2
```

**Comment by Corben78** [comment]
Alright, I got it working!

tl;dr: get ceg exe files via wine, ~~install xact.~~ (see https://github.com/ValveSoftware/Proton/issues/530#issuecomment-433696430)

Long:
* download AvP via windows steam through wine
* copy/move `steamapps/common/Aliens vs Predator` files and `steamapps/appmanifest_10680.acf` from the wine prefix to your Linux steam library folder
* install xact
* ~~set `xaudio2_5`to native or native/builtin~~

This has to be done on the same machine, as the ceg drm generates executables for your machine. Make backups of these two files:
```bash
cp AvP.exe{,.bak}
cp AvP_DX11.exe{,.bak}
```
If you verify the integrity of the gamefiles, those to .exe files will be deleted, and redownloaded when starting AvP, but then the start will fail as ceg download seems not to work with the Linux steam client.

Start AvP so it creates the proton prefix. It will work, but AvP will crash after a short while. Don't wonder, there is a small black box on the top left using DX9 or a small window using DX11 before the splash screen shows up. The crash can be fixed via this override:

To set the xaudio2_5 override, e.g. use your wine install:
```bash
WINEPREFIX=<SteamLibrary>/steamapps/compatdata/10680/pfx /opt/wine-staging/bin/winecfg
```
Add xaudio2_5 to the overrides, native or native/builtin.

edit: seems to be enough just to install xact, it's not necessary to set the override:
```bash
WINEPREFIX=<SteamLibrary>/steamapps/compatdata/10680/pfx WINE=<SteamLibrary>/steamapps/common/Proton\ 3.16/dist/bin/wine64 winetricks xact
```

I played it now for a while, and it doesn't seem to crash anymore. Works beautifully with DX9 as well as DX11. Tested with proton 3.16-3.

**Comment by Zlopez** [comment]
Doesn't start for me. 

Here is the Proton log: 
[steam-10680.log](https://github.com/ValveSoftware/Proton/files/2511615/steam-10680.log)

And my specs:
https://gist.github.com/Zlopez/1433022f5f75aebc3155a622997b2997

**Comment by R3D9477** [comment]
@Corben78 I can confirm it, doesn't start for me, too.
Linux Mint 19 x64

Computer Information:
    Manufacturer:  Unknown
    Model:  Unknown
    Form Factor: Desktop
    No Touch Input Detected

Processor Information:
    CPU Vendor:  GenuineIntel
    CPU Brand:         Intel(R) Core(TM) i5-2520M CPU @ 2.50GHz
    CPU Family:  0x6
    CPU Model:  0x2a
    CPU Stepping:  0x7
    CPU Type:  0x0
    Speed:  3200 Mhz
    4 logical processors
    2 physical processors
    HyperThreading:  Supported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Unsupported
    SSE41:  Supported
    SSE42:  Supported
    AES:  Supported
    AVX:  Supported
    CMPXCHG16B:  Supported
    LAHF/SAHF:  Supported
    PrefetchW:  Unsupported

Operating System Version:
    Linux Mint 19 Tara (64 bit)
    Kernel Name:  Linux
    Kernel Version:  4.15.0-38-generic
    X Server Vendor:  The X.Org Foundation
    X Server Release:  11906000
    X Window Manager:  Compiz
    Steam Runtime Version:  steam-runtime-beta-release_2018-09-03

Video Card:
    Driver:  Intel Open Source Technology Center Mesa DRI Intel(R) Sandybridge Mobile x86/MMX/SSE2
    Driver Version:  3.0 Mesa 18.0.5
    OpenGL Version: 3.0
    Desktop Color Depth: 24 bits per pixel
    Monitor Refresh Rate: 59 Hz
    VendorID:  0x8086
    DeviceID:  0x126
    Revision Not Detected
    Number of Monitors:  1
    Number of Logical Video Cards:  1
    Primary Display Resolution:  1366 x 768
    Desktop Resolution: 1366 x 768
    Primary Display Size: 13.54" x 7.64" (15.51" diag)
                                            34.4cm x 19.4cm (39.4cm diag)
    Primary VRAM Not Detected

Sound card:
    Audio device: Conexant CX20590

Memory:
    RAM:  7859 Mb

Miscellaneous:
    UI Language:  English
    LANG:  en_US.UTF-8
    Total Hard Disk Space Available:  55071 Mb
    Largest Free Hard Disk Block:  29364 Mb
    VR Headset: None detected

Recent Failure Reports:

**Comment by Corben78** [comment]
This is really odd (pun intended).

As said, it does only start at all if you copy the executables (AvP.exe, AvP_DX11.exe) from a windows installation (e.g. wine, maybe even windows) from the same machine to the Linux AvP installation folder. The Steam Linux client (as well as steamcmd, both win and lin) doesn't download a working executable.

Now the game is crashing for me, e.g. when pulling the first lever in the Marine campaign, or killing the first Alien in survivor mode (probably the quickest way to test) on odd days, but not on even days. I tried this by just changing the date on my machine and restarting the game. E.g. crashing on 27th, working on 28th, crashing on 29th, working on 30th, crashing on 31th, also crashing on the 1st... (I didn't test more yet). It's the same behavior on two machines for me (Desktop and Notebook).

So maybe this is a date-dependent initializer, as iirc when it was working in proton, it didn't crash after passing midnight as long as I didn't restarting the game.

Strange is though, that this does not happen with the wine version. Here it's working each day.

Also, it's not needed to install xact via winetricks. On the first time setup, the game wants to open a ReadMe.txt. But in proton a file called ' readme.txt' is wanted, which doesn't exist. So if you let the opened notepad create that file and close notepad then, the first time setup will finish correctly and install the xaudio dlls. If you close notepad without creating that file, the first time setup seems to be interrupted, and you don't have any audio. If you start the game a second time, the setup continues, and then you have sound. So winetricks xact is not needed at all.

So the symptom is now, that it's working on every second day. The cause might be a completely different though.

edit: workaround to determine the date and set the next even day and then start the game:
* install faketime libfaketime:i386
* set launch parameter: `faketime -f +$(($(date +%d) % 2))d %command%`

edit:
* set launch parameter: `D=$(date +%d); [ $D == 31 ] && D=2 || D=$(($D % 2)); faketime -f +"$D"d %command%`
with this, it will also work on the 31st of the month

edit:
K, testing today (31st of october) and it crashes all the time, looks like faketime doesn't help, but changing my system time helped. At least for the first Alien kill in survivor mode. But Marine and Predator tutorial crashed...

edit:
Just tested again (2nd of november) and everything works! The three easy situations to quickly reach for a test are:
* Marine tutorial, pull the lever after finding the first dead civilian
* Predator tutorial, look at the third Predator
* Survival, first Alien kill

This all worked today flawless.

**Comment by tr37ion** [comment]
Works without issues. Started DX11 version and played some minutes into the Marine mission.

**Comment by Corben78** [comment]
Looks like the behaviour changed in 2019. Yesterday (3rd jan) we were playing multiplayer, and when I was killing my first enemy (be it killing an Alien as Marine, or as Marine killing an Alien) the game crashed.

It did not crash with the previous triggers, like pulling the first lever in the Marine campaign, so this issue might have been fixed in Proton 3.16-5 or 3.16-6 with @flibitijibo's FAudio implementation.

Today (4th jan) I was testing again, and it did not crash on the first Alien kill. So I guess in 2019 we can play AvP on even days and not on odd days ;)

edit: you still need the correct ceg executables from a windows or wine install of the same machine, or it won't start at all.

edit: Looks like I was a bit confused, it's still as it was last year. Working on even days, not on odd days. That did not change. But my faketime command worked today (5th jan).

**Comment by Corben78** [comment]
There seems a regression with mouse movement since Proton 4.2.

Though you can get AvP working with above workarounds (getting a ceg protected executable from a windows/wine install of the same machine, and faking the date so it doesn't crash at certain trigger points), the mouse movement is inverted for a certain movement distance.
So if you change the direction of your movement the mouse still continues or starts in the other direction for a while, making the character unsteerable and thus the game unplayable. This affects all directions: left, right, up, down.

Also quitting the game doesn't work any more, the game locks up on quit and has to be killed.
It's still fine (regarding mouse movement and quitting the game) and playable with proton 3.16-9, but not with later Versions (tested with 4.2-9 and 4.11-7).
Haven't seen this weird mouse behaviour with other games so far. 

(A kind of similar issue has been with Subnautica in a former Proton version, but only concerning the down movement and way less notable, was fixed in Subnautica experiemental branch or in a later Proton version. Atm I play Subnautica with 4.11 flawlessly).

**Comment by R3D9477** [comment]
Still can't laucnh the game.
Currently I'm on KDE Neon (Ubuntu 18.04) with Nvidia.
I've tried with both DX9 and DX11.

## System Information
- GPU: GeForce 930MX
- Driver/LLVM version: nvidia 435.21
- Kernel version: 5.0.0-36-generic
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/r3d9u11/8ab0a223b9fbf00cc93b85e5c2d76ce6
- Proton version: 4.11-9

[steam-10680.log](https://github.com/ValveSoftware/Proton/files/3911047/steam-10680.log)

**Comment by andyholst** [comment]
I had to install the Alien vs Predator game in a **Windows Steam version with Wine to get the proper generated AvP_DX11.exe AvP.exe** files and copy those to **~/.steam/steam/steamapps/common/Aliens\ vs\ Predator/** to be able to execute the game perfectly with Steam Play compatibility tool Proton.

System information

```
Proton version: 4.11-11
System:    Host: asimov-MacBookPro Kernel: 5.4.6-050406-generic x86_64 bits: 64 compiler: gcc 
           v: 9.2.1 Desktop: Cinnamon 4.4.6 wm: muffin dm: LightDM Distro: Linux Mint 19.3 Tricia 
           base: Ubuntu 18.04 bionic 
Machine:   Type: Laptop System: Apple product: MacBookPro13,3 v: 1.0 serial: <filter> Chassis: 
           type: 9 v: Mac-A5C67F76ED83108C serial: <filter> 
           Mobo: Apple model: Mac-A5C67F76ED83108C v: MacBookPro13,3 serial: <filter> UEFI: Apple 
           v: 263.0.0.0.0 date: 10/30/2019 
Battery:   ID-1: BAT0 charge: 53.4 Wh condition: 54.5/76.7 Wh (71%) volts: 12.7/11.5 
           model: SMP bq20z451 serial: N/A status: Full 
           Device-1: hidpp_battery_0 model: Logitech Wireless Keyboard serial: <filter> 
           charge: 55% status: Discharging 
CPU:       Topology: Quad Core model: Intel Core i7-6920HQ bits: 64 type: MT MCP arch: Skylake-S 
           rev: 3 L2 cache: 8192 KiB 
           flags: lm nx pae sse sse2 sse3 sse4_1 sse4_2 ssse3 vmx bogomips: 46398 
           Speed: 900 MHz min/max: 800/3800 MHz Core speeds (MHz): 1: 900 2: 900 3: 900 4: 900 
           5: 900 6: 900 7: 900 8: 900 
Graphics:  Device-1: AMD Baffin [Radeon RX 460/560D / Pro 450/455/460/555/560] vendor: Apple 
           driver: amdgpu v: kernel bus ID: 01:00.0 chip ID: 1002:67ef 
           Display: x11 server: X.Org 1.20.4 driver: amdgpu,ati unloaded: fbdev,modesetting,vesa 
           resolution: 1920x1080~60Hz, 1920x1080~60Hz 
           OpenGL: 
           renderer: AMD Radeon RX Graphics (POLARIS11 DRM 3.35.0 5.4.6-050406-generic LLVM 7.1.0) 
           v: 4.5 Mesa 18.3.0-rc4 direct render: Yes 
Audio:     Device-1: Intel 100 Series/C230 Series Family HD Audio driver: snd_hda_intel v: kernel 
           bus ID: 00:1f.3 chip ID: 8086:a170 
           Device-2: AMD driver: snd_hda_intel v: kernel bus ID: 01:00.1 chip ID: 1002:aae0 
           Sound Server: ALSA v: k5.4.6-050406-generic 
Network:   Device-1: Broadcom and subsidiaries BCM43602 802.11ac Wireless LAN SoC vendor: Apple 
           driver: brcmfmac v: kernel port: 3000 bus ID: 03:00.0 chip ID: 14e4:43ba 
           IF: wlp3s0 state: up mac: <filter> 
           IF-ID-1: docker0 state: down mac: <filter> 
Drives:    Local Storage: total: 465.92 GiB used: 131.02 GiB (28.1%) 
           ID-1: /dev/nvme0n1 vendor: Apple model: SSD SM0512L size: 465.92 GiB speed: 31.6 Gb/s 
           lanes: 4 serial: <filter> 
Partition: ID-1: / size: 455.46 GiB used: 65.31 GiB (14.3%) fs: ext4 dev: /dev/dm-1 
           ID-2: /boot size: 704.5 MiB used: 319.2 MiB (45.3%) fs: ext4 dev: /dev/nvme0n1p2 
           ID-3: swap-1 size: 979.5 MiB used: 79.5 MiB (8.1%) fs: swap dev: /dev/dm-3 
Sensors:   System Temperatures: cpu: 59.0 C mobo: N/A 
           Fan Speeds (RPM): N/A 
Repos:     No active apt repos in: /etc/apt/sources.list 
           Active apt repos in: /etc/apt/sources.list.d/additional-repositories.list 
           1: deb [arch=amd64] https: //download.docker.com/linux/ubuntu bionic stable
           Active apt repos in: /etc/apt/sources.list.d/amdgpu-pro-local.list 
           1: deb [ trusted=yes ] file: /var/opt/amdgpu-pro-local/ ./
           Active apt repos in: /etc/apt/sources.list.d/graphics-drivers-ppa-bionic.list 
           1: deb http: //ppa.launchpad.net/graphics-drivers/ppa/ubuntu bionic main
           Active apt repos in: /etc/apt/sources.list.d/lutris-team-lutris-bionic.list 
           1: deb http: //ppa.launchpad.net/lutris-team/lutris/ubuntu bionic main
           Active apt repos in: /etc/apt/sources.list.d/nodesource.list 
           1: deb https: //deb.nodesource.com/node_10.x bionic main
           2: deb-src https: //deb.nodesource.com/node_10.x bionic main
           Active apt repos in: /etc/apt/sources.list.d/official-package-repositories.list 
           1: deb http: //ftp.acc.umu.se/mirror/linuxmint.com/packages tricia main upstream import backport
           2: deb http: //archive.ubuntu.com/ubuntu bionic main restricted universe multiverse
           3: deb http: //archive.ubuntu.com/ubuntu bionic-updates main restricted universe multiverse
           4: deb http: //archive.ubuntu.com/ubuntu bionic-backports main restricted universe multiverse
           5: deb http: //security.ubuntu.com/ubuntu/ bionic-security main restricted universe multiverse
           6: deb http: //archive.canonical.com/ubuntu/ bionic partner
           Active apt repos in: /etc/apt/sources.list.d/skype-stable.list 
           1: deb [arch=amd64] https: //repo.skype.com/deb stable main
Info:      Processes: 299 Uptime: 18h 42m Memory: 15.54 GiB used: 2.59 GiB (16.7%) Init: systemd 
           v: 237 runlevel: 5 Compilers: gcc: 7.4.0 alt: 7 Client: Unknown python3.6 client 
           inxi: 3.0.32 
```

**Comment by Corben78** [comment]
The strange mouse behaviour mentioned in https://github.com/ValveSoftware/Proton/issues/530#issuecomment-544172027 seems to be fixed with in Proton 5.0 (tested with 5.0-3)

**Comment by Soulsuke** [comment]
The installation process still doesn't work, so I've hard to copy the exe files from a Wine Steam install.

I still get consistent crashes on both Predator and Marine tutorials. So far survival and Alien story mode seem to be working (I've tried the fakedate workaround but it doesn't work). Also, the process hands after exiting the application, and has to be manually killed.

I'm currently on Arch 64 and I've tried all the available Proton versions (from 5.0-5 to 3.7-8).

I attached a log of the crash during the Predator's tutorial.

[steam-10680.log](https://github.com/ValveSoftware/Proton/files/4444730/steam-10680.log)

**Comment by R3D9477** [comment]
Please, fix this annoying bug.

Just replace AvP.exe in launcher for Linux with Windows-version of this file (as noted here: https://www.protondb.com/app/10680#8k3i4GHRsF )

I don't know what the difference between versions of .exe file for Linux and Windows, but from from windows-version works fine on Linux. So installation of AVP in Linux is really pain and it can be fixed simple, as I understand right 😭😭😭

**Comment by Corben78** [comment]
Game works now with latest Steam Client beta and Proton experimental 6.3-20211027.

I played through some of the trigger points, where the game was crashing before and had no issues.

The game just loses audio when tabbing out (also tabbing out with Steam overlay open).

**Comment by dpanter** [comment]
Game seems to work fine indeed with beta client and latest Proton experimental.
I worked around the alt-tab sound issue with windowed mode, launch option `-subwindow` and simply toggling fullscreen with KDE/Kwin (default hotkey F11). Game still captures mouse in gameplay and alt-tab doesn't break sound anymore.

**Comment by AwesamLinux** [comment]
# Compatibility Report
- Name of the game with compatibility issues: Aliens vs. Predator
- Steam AppID of the game: 10680

## System Information
- GPU: AMD Radeon RX 5700 
- Driver/LLVM version: Mesa 21.2.4 - kisak-mesa PPA
- Kernel version:  5.11.0-38-generic
- Proton version:  proton experimental-6.3-20211027

## Symptoms 

- Game launches minimized to taskbar, have to manually click or alt+tab up the game
- No sound when the game is fullscreen, in windowed mode works fine

[steam-10680.log](https://github.com/ValveSoftware/Proton/files/7443646/steam-10680.log)
[system_info.txt](https://github.com/ValveSoftware/Proton/files/7443647/system_info.txt)

**Comment by thaewrapt** [comment]
Playable on latest Beta + Experimental, also starts in fullscreen on my `fluxbox` WM (same as with Colonial Marines).

**Comment by Zlopez** [comment]
With latest beta + experimental I have issue with mouse in the menu  (the mouse is visible, but the menu options are not reacting to it) and the sound issue mentioned earlier.

Specs: https://gist.github.com/Zlopez/1433022f5f75aebc3155a622997b2997

**Comment by R3D9477** [comment]
Another one problem with AvP (now installation is failing):

`The Steam servers are too busy to handle your request for Aliens vs. Predator. Error Code (55)`

Have tried several times.

**Comment by dpanter** [comment]
This issue is solved, time to close.

**Comment by BLaDZer** [comment]
Game crashing in the first predator mission
[steam-10680.log](https://github.com/ValveSoftware/Proton/files/13060841/steam-10680.log)

**Comment by kisak-valve** [comment]
Controls not working

Issue transferred from https://github.com/ValveSoftware/steam-for-linux/issues/12479.
@navrajaytan41-hub posted on 2025-11-24T21:00:40:

I recently purchased aliens vs predator and when i changed the controls it hasn’t worked

**Comment by Raxyz0** [comment]
Tried everything in this thread and some fixes suggested on ProtonDB. Nothing worked, game outright fails to launch every time. Standard launch, DX9, DX11, Proton 5, Proton 8, Proton 3, Experimental and GE. Just outright fails to work.

I also couldn't find a log file for my game other than this one. Let me know if there's anything I can provide that may help understand what's happening.

[DirectX.log](https://github.com/user-attachments/files/31513180/DirectX.log)

**Comment by kisak-valve** [comment]
Hello @Raxyz0, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, please copy the contents of Steam Runtime Diagnostics from Steam (`Steam` -> `Help` -> `Steam Runtime Diagnostics`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by Raxyz0** [comment]
Thanks for the quick reply. Here they are:

[steam-10680.log](https://github.com/user-attachments/files/31517061/steam-10680.log)
[Steam Runtime Diagnostic](https://gist.github.com/Raxyz0/92b37ec74ac5515554a0cba6755e0659)

**Comment by kisak-valve** [comment]
Blind guess, can you opt out of Steam's steamrt3 beta and check if that has an effect?

**Comment by Raxyz0** [comment]
> Blind guess, can you opt out of Steam's steamrt3 beta and check if that has an effect?

Spot on. I turned that on as I was trying to get Remote Play Together to work and I forgot I was even on that branch.
I've disabled that option, restarted Steam and it's now working with no additional tweaks. Thank you!
